Examples/KeyboardSupport-iOSExample/ViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ViewController.swift
+//  KeyboardSupport-iOSExample
+//
+//  Created by Earl Gaspard on 7/30/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import UIKit
+import KeyboardSupport
+
+class ViewController: UIViewController, KeyboardRespondable {
+
+    // IBOutlets
+    @IBOutlet private var scrollView: UIScrollView!
+    @IBOutlet private var textField1: UITextField!
+    @IBOutlet private var textField2: UITextField!
+    @IBOutlet private var textField3: UITextField!
+    @IBOutlet private var textView: UITextView!
+    
+    // KeyboardScrollable
+    var keyboardScrollableScrollView: UIScrollView?
+    var keyboardWillShowObserver: NSObjectProtocol?
+    var keyboardWillHideObserver: NSObjectProtocol?
+    
+    // KeyboardNavigator
+    private var keyboardNavigator: KeyboardNavigator?
+    
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        textView.delegate = self
+        
+        // KeyboardDismissable setup
+        setupKeyboardDismissalView()
+        
+        // KeyboardScrollable setup
+        keyboardScrollableScrollView = scrollView
+        setupKeyboardObservers()
```

Comment:
After running the sample app, I noticed that the "keyboardWillShow/Hide" log messages are printed twice every time the keyboard appears or disappears. It's because we're calling `setupKeyboardObservers()` twice - once here in `viewDidLoad()` and again in `viewWillAppear()`. We need to remove the call here and instead rely only on `viewWillAppear/Disappear()`.

Reply:
Fixed